Explorer/Assets/DCL/PluginSystem/PluginContainerExtensions.cs

```diff
@@ -23,6 +25,47 @@ CancellationToken ct
             return (plugin, true);
         }
 
+        public static async UniTask<(TPlugin plugin, bool success)> InitializePluginWithAnalyticsAsync<TPlugin>(
+            this IPluginSettingsContainer pluginSettingsContainer,
+            TPlugin plugin,
+            IAnalyticsController analytics,
+            CancellationToken ct
+        ) where TPlugin: class, IDCLPlugin
+        {
+            string pluginName = plugin.GetType().Name;
+
+            analytics.Track(AnalyticsEvents.General.PLUGINS_INIT, new JsonObject
```

Comment:
For readability I would remove the duplication into a local Track(string status) function:

`    void Track(string s) =>
        analytics.Track(AnalyticsEvents.General.PLUGINS_INIT, new JsonObject { { "plugin", pluginName }, { "status", s } });`